eth/v1/debug.proto

```diff
@@ -0,0 +1,111 @@
+// Copyright 2020 Prysmatic Labs.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+syntax = "proto3";
+
+package ethereum.eth.v1;
+
+option csharp_namespace = "Ethereum.Eth.v1";
+option go_package = "github.com/prysmaticlabs/ethereumapis/eth/v1";
+option java_multiple_files = true;
+option java_outer_classname = "DebugProto";
+option java_package = "org.ethereum.eth.v1";
+option php_namespace = "Ethereum\\Eth\\v1";
+
+import "google/protobuf/empty.proto";
+import "google/protobuf/any.proto";
+
+import "eth/v1/attestation.proto";
+import "eth/v1/beacon_states.proto";
+import "eth/v1/beacon_blocks.proto";
+import "eth/v1/beacon_block.proto";
+import "eth/v1/validator.proto";
+
+// Beacon chain debug API
+//
+// The beacon chain debug API is a set of endpoints to debug chain and shouldn't be exposed publicly.
+service Debug {
+  // GetBeaconState returns full BeaconState object for given stateId.
+  rpc GetBeaconState(StateRequest) returns (BeaconStateResponse) {
+    option (google.api.http) = {get: "/eth/v1/debug/beacon/states/{state_id}"};
+  }
+
+  // ListForkChoiceHeads retrieves all possible chain heads (leaves of fork choice tree).
+  rpc ListForkChoiceHeads(google.protobuf.Empty) returns (ForkChoiceHeadsResponse) {
+    option (google.api.http) = {get: "/eth/v1/debug/beacon/heads"};
+  }
+}
+
+message ForkChoiceHeadsResponse {
+  repeated ForkChoiceHead data = 1;
+}
+
+message ForkChoiceHead {
+  bytes root = 1 [(gogoproto.moretags) = "ssz-size:\"32\""];
+  uint64 slot = 2;
+}
+
+message BeaconStateResponse {
+  BeaconStateContainer data = 1;
+}
+
+message BeaconStateContainer {
```

Comment:
This isn't really a container, it's the beacon state itself.
```suggestion
message BeaconStateResponse {
  BeaconState data = 1;
}

message BeaconState {
```

Reply:
Can't do this rename because of how the API is structured, will make a follow up PR.